tests/suite-unit/PluginFlyvemdmCommon.php

```diff
@@ -137,7 +137,7 @@ public function testGetMax() {
     * Used as provider for start/ends with methods
     * @return array
     */
-   protected function stringProvider() {
+   protected function providerStartsAndEndWith() {
```

Comment:
it is more generic to have an other provider for testEndsWith. The goal is to force a matching between a provider and a test.

```php 
public function providderEndsWith() {
   return $this->providerStartsWith();
}
```

Reply:
That's why it was only providerString, because it is used for both test

Reply:
but we loose the relationship between name of provider and name of test